File: cs/src/AutoDiff/Compiled/Sigmoid.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AutoDiff.Compiled
{
	class Sigmoid : TapeElement
    {
        public int Arg;
        public int Mid;
		public double Steepness;

        public override void Accept(ITapeVisitor visitor)
        {
            visitor.Visit(this);
        }
	}
}

File: java/src/main/java/AutoDiff/Compiled/Sigmoid.java
//
// Translated by CS2J (http://www.cs2j.com): 12.11.2016 09:40:32
//

package AutoDiff.Compiled;

public class Sigmoid  extends TapeElement
{
    public int Arg;
    public int Mid;
    public double Steepness;
    public void accept(ITapeVisitor visitor) throws Exception {
        visitor.visit(this);
    }

}


